src/Common/Thermodynamics/relations.jl

```diff
@@ -1532,7 +1543,7 @@ function saturation_adjustment_q_tot_θ_liq_ice_given_pressure(
     if unsaturated && T_1 > _T_min
         return T_1
     else
-        T_2 = air_temperature_from_liquid_ice_pottemp(
+        T_2 = air_temperature_given_θρq(
```

Comment:
My eyes are bad. But maybe I see rho in function name but pressure in argument list?

Reply:
If only the compiler could check units of our arguments. This would be a compiler error!

Reply:
Good catch! Looks like a bug to me! Look at that, including `ρ` in the name came in handy!

Reply:
Wow, not sure how that's been passing CI 🤔. 😮 This branch is potentially hit in BOMEX/Dycoms initialization!

Reply:
Updated. 😬 Alright, merge merge merge! 😆

Reply:
Maybe this is responsible for the strange spike in `q_tot` in the BOMEX init. Could be impacting EDMF, too! @yairchn

Reply:
Ah, actually, I think we want to leave pressure but call `air_temperature_given_θpq`. Fixing..

Reply:
Ah, good news, this was only used for computing a temperature bound, so not terrible.